gprofiler/spark/spark_collector.py

```diff
@@ -672,6 +749,19 @@ def _guess_yarn_resource_manager_webapp_address(self, resource_manager_process:
             host_name = self._get_yarn_host_name(resource_manager_process)
             return host_name + ":8088"
 
+    def _guess_standalone_master_webapp_address(self, process: psutil.Process) -> str:
+        """
+        Selects the master address for a standalone cluster.
+        Uses master_address if given.
+        """
+        if self._master_address:
+            return self._master_address
+        else:
+            master_process_args = process.cmdline()
+            master_ip = master_process_args[master_process_args.index("--host") + 1]
+            master_port = master_process_args[master_process_args.index("--webui-port") + 1]
+            return f"{str(master_ip)}:{str(master_port)}"
+
```

Comment:
Please move the function in place of the original so we can get a diff of the implementation.